VirtoCommerce.InventoryModule.Web/Controllers/Api/InventoryModuleController.cs

```diff
@@ -71,7 +83,21 @@ public IHttpActionResult SaveFulfillmentCenter(FulfillmentCenter center)
         }
 
         /// <summary>
-        /// Delete  fulfillment centers registered in the system
+        /// Bulk save fulfillment centers
+        /// </summary>
+        /// <param name="centers">fulfillment centers</param>
+        [HttpPut]
```

Comment:
Done.

Reply:
Let's make PUT fulfillmentcenters/batch (as in products)
![image](https://user-images.githubusercontent.com/15610352/69522934-70c0ba80-0f84-11ea-91cd-873d6751e7df.png)
Because now we have POST & PUT fulfillmentcenters/plenty with an unclear difference by the HTTP verb.

Reply:
done